developerportal/templates/organisms/newsletter-signup.html

```diff
@@ -27,7 +27,7 @@ <h2>Discover the best of web development</h2>
           <p class="newsletter-privacy-wrapper">
             <label for="newsletter-privacy" class="mzp-u-inline">
               <input type="checkbox" name="privacy" id="newsletter-privacy" required aria-required="true">
-              I'm okay with Mozilla handling my info as explained in this <a class="privacy-link" href="https://www.mozilla.org/privacy/websites/" target="_blank" rel="noopener noreferrer">Privacy Notice</a>.
+              I'm okay with Mozilla handling my info as explained in this <a class="privacy-link" href="https://www.mozilla.org/privacy/websites/" target="_blank" rel="noopener">Privacy Notice</a>.
```

Comment:
@stevejalim Is there a specific reason the link to the privacy policy is the odd one out?

Reply:
Ah! because this is one of the rare instances that we *do* know that we are linking to an external site we are comfortable endorsing.

Reply:
My logic was: given we control both sides of the interaction (ie, link from a Mozilla property to another Mozilla property) there's no risk from including the referrer and it could be useful to analytics for the target side. 

However, I'm happy to change it! 😄

Reply:
Nope, I 100% agree. Took me a second to put two and two together 😄